Tests/Microsoft.AppCenter.Crashes.Test.Windows/packages.config

```diff
@@ -20,18 +20,18 @@
   <package id="System.Diagnostics.Tracing" version="4.1.0" targetFramework="net452" />
   <package id="System.Globalization" version="4.0.11" targetFramework="net452" />
   <package id="System.IO" version="4.1.0" targetFramework="net452" />
-  <package id="System.IO.Compression" version="4.1.0" targetFramework="net452" />
+  <package id="System.IO.Compression" version="4.1.0" targetFramework="net452" requireReinstallation="true" />
```

Comment:
what is that new flag? also I believe we need to update targetFramework here.

Reply:
from UI..
I'll recheck it